### PR TITLE
chore(ctb): Remove some todos in messenger and portal

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

1. Removed some todos, I'll provide the rationale in my self-review. 
2. Made error messages conform to style in other contracts.

### CODE REVIEW

File: packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol [hunk 1]
```diff
@@ -261,26 +264,29 @@ abstract contract CrossDomainMessenger is
 
         if (_isSystemMessageSender()) {
             // Should never happen.
-            require(msg.value == _value, "Mismatched message value.");
+            require(msg.value == _value, "CrossDomainMessenger: Mismatched message value.");
         } else {
-            // TODO(tynes): could require that msg.value == 0 here
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol:266]: Hmmm, I should add a test for this actually.

File: packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol [hunk 2]
```diff
@@ -261,26 +264,29 @@ abstract contract CrossDomainMessenger is
...
-            require(receivedMessages[versionedHash], "Message cannot be replayed.");
+            require(msg.value == 0, "CrossDomainMessenger: Cannot relay messages with value");
+            require(
+                receivedMessages[versionedHash],
+                "CrossDomainMessenger: Message cannot be replayed."
+            );
         }
 
-        // TODO: Should blocking happen on sending or receiving side?
-        // TODO: Should this just return with an event instead of reverting?
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol:272]: Either would be secure, but I think reverting on an invalid action is much more standard.

File: packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol [hunk 3]
```diff
@@ -16,7 +16,6 @@ import { OptimismMintableERC20 } from "../universal/OptimismMintableERC20.sol";
  *         Examples of some token types that may not be properly supported by this contract include,
  *         but are not limited to: tokens with transfer fees, rebasing tokens, and
  *         tokens with blocklists.
- *         TODO: ensure that this has 1:1 backwards compatibility
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol:19]: I assume that this just refers to function sigs and storage layouts, which we're already tracking.

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
Gas snapshot out of date

**Pull request merged**